pkg/crypto.go

```diff
@@ -70,5 +80,32 @@ func (c Crypto) Setup(hmacFile string, masterFile string) (bool, error) {
 }
 
 func (c Crypto) Protect(value string) (string, error) {
-	return value, nil // TODO implement this
+	var form = map[string]any{"datum": value}
+	log.Infof("%s > Protecting text using crypto.", c.instance.ID())
+	requestFormData := c.instance.http.RequestFormData(form)
+	response, err := requestFormData.Post(EncryptJsonPath)
+
+	if err != nil {
+		return "", fmt.Errorf("%s > Error while encrypting text using crypto: %s", c.instance.ID(), err)
```

Comment:
Do not capitalize text

Reply:
Why this starts with "error while..." but other msgs with "cannot". Align the style used

Reply:
I missed this error :)